packages/thumbprint-react/components/Calendar/index.tsx

```diff
@@ -33,13 +16,16 @@ function throwError(message: string): void {
 }
 
 export function normaliseValue(value: CalendarProps['value']): Date[] {
-    if (value === null) {
+    if (!value) {
         return [];
     }
 
-    const valueArr: DateIsh[] = castArray<DateIsh>(value);
+    if (!isArray(value)) {
+        const valueArr: DateIsh[] = [value];
+        return map(valueArr, d => new Date(d));
```

Comment:
since this is guaranteed to be a single value, we can simplify these two lines to

`return [new Date(value)]`

that handles converting the value to a date, and wrapping in an array, in a single line

Reply:
Oppps! I missed it! Updated!